src/libasr/codegen/asr_to_llvm.cpp

```diff
@@ -1010,6 +1011,7 @@ class ASRToLLVMVisitor : public ASR::BaseVisitor<ASRToLLVMVisitor>
         complex_type_8 = llvm::StructType::create(context, els_8, "complex_8");
         complex_type_4_ptr = llvm::StructType::create(context, els_4_ptr, "complex_4_ptr");
         complex_type_8_ptr = llvm::StructType::create(context, els_8_ptr, "complex_8_ptr");
+        tuple_type = llvm::StructType::create(context, els_8, "tuple");
```

Comment:
`els_8` is a two element vector of pointer to double types. So that means all the tuples would look like `{double*, double*}`?